docs/content/direct/get-started.md

```diff
@@ -1,32 +1,23 @@
 # KubeStellar Quickstart Setup
 
-This Quick Start outlines step 1 and shows a concrete example of steps 2--7 in the [Installation and Usage outline](usage-basics.md). In this example you will create three new `kind` clusters to serve as your KubeFlex hosting cluster and two WECs.
+This Quick Start outlines step 1, shows a concrete example of steps 2--7 in the [Installation and Usage outline](usage-basics.md), and forwards you to one example of the remaining steps. In this example you will create three new `kind` clusters to serve as your KubeFlex hosting cluster and two WECs.
 
   1. Before you begin, prepare your system (get the software prerequisites)
   2. Create the KubeFlex hosting cluster and Kubestellar core components
   3. Create and register two WECs.
+  4. Use KubeStellar to distribute a Deployment object to the two WECs.
 
 ---
 ## Before You Begin
 
+The following command will check for the prerequisites that you will need for this quickstart. See [the prerequisites doc](pre-reqs.md) for more details.
 
-{%
-    include-markdown "pre-reqs.md"
-    rewrite-relative-urls=true
-    start="<!-- begin software prerequisites -->"
-    end="Additional Software For Running"
-    heading-offset=1
-%}
----
-{%
-    include-markdown "pre-reqs.md"
-    rewrite-relative-urls=true
-    heading-offset=2
-    start="<!-- start tag for check script  include -->"
-    end="<!-- end tag for check-prereq script -->"
-%}
+```shell
+bash <(curl https://raw.githubusercontent.com/kubestellar/kubestellar/v{{ config.ks_latest_regular_release }}/hack/check_pre_req.sh) kflex ocm helm kubectl docker kind
+```
 
----
+This quickstart uses [kind](https://kind.sigs.k8s.io/) to create three Kubernetes cluster on your machine.
+Note that `kind` does not support three or more concurrent clusters unless you raise some limits as described in this `kind` "known issue": [Pod errors due to “too many open files”](https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files).
 
 ### Delete debris from preivous trials
```

Comment:
Not part of this PR - but I would move that to a special section at the end "Cleanup"

Reply:
I think that putting the cleanup at the end is bad for two reasons.

Tearing down the state as soon as the user finishes going through the written recipe is bad because it deprives the user of the ability to examine the state that the recipe established and do further experiments with it.

Granted, the user might figure that out and put off the cleanup. In this case, it is not unlikely that the user will forget about the cleanup by the time they go to start over again. I have done this, and I know that others have done this.

Reply:
Sounds reasonable.  My comment was mainly about the sections and titles .  See the second comment which basically covers the same thing.
I think that "Delete debris from previous trials"  and "Before you begin, prepare your system (get the software prerequisites)"  can be simplified ...

Reply:
Revised.